tests/model_registry/conftest.py

```
@@ -61,128 +61,171 @@
+            template=get_model_registry_deployment_template_dict(
+                secret_name=model_registry_db_secret.name, resource_name=DB_RESOURCES_NAME
+            ),
+            wait_for_resource=True,
+            teardown=teardown_resources,
+        ) as mr_db_deployment:
+            mr_db_deployment.wait_for_replicas(deployed=True)
+            yield mr_db_deployment
 
 
 @pytest.fixture(scope="class")
 def model_registry_instance(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)